### ISSUE TITLE
Weird icons on MacOSx Electron

### ISSUE THREAD

**Issue by briosheje** [vmware/clarity#969]
**Select one ...**  (check one with "x")
```
[x] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior

I would expect the icons to just work.

### Actual behavior

Icons are... weird?

### Reproduction of behavior

* Include a link to the reproduction scenario you created by forking one of the Clarity Plunker Templates:

Can't reproduce on a plunkr since it's tested on electron and chromium.

Icons act in this way:

![schermata 2017-05-24 alle 22 28 52](https://cloud.githubusercontent.com/assets/1898880/26424069/3e2c2b0c-40d0-11e7-8246-d8af24dc0d8c.png)

The whole application is boundled through Webpack. All the clarity files are included through a "vendor" file and a "polyfills" file:

polyfills.ts
```
/**
 * Clarity stuff
 */
import "mutationobserver-shim/dist/mutationobserver.min.js";
import "@webcomponents/custom-elements/custom-elements.min.js";
```

vendor.ts

```
/**
 * Clarity stuff
 */
import "clarity-ui/clarity-ui.min.css";
import "clarity-icons/clarity-icons.min.css";
import "clarity-icons";
```

Everything seems to be compiled properly and everything is working properly, the icons are being rendered in weird ways, not sure whether this is supposed to happen because it's chromium (with electron), but it's quite weird anyway.

Am I missing something?

### Environment details

* **Angular version:** 4.0.0

* **Clarity version:** 0.9.5

* **OS and version:** MacOSx Sierra 10.12.4

* **Browser:** [ Chrome | Chromium (electron) ]

**Comment by mathisscott** [comment]
@briosheje 
Yeah, that looks really odd. We will have to investigate. It's really hard to tell what's happening there. Possibly related to something in chromium. It doesn't look like it is sizing or rendering the SVG properly.

**Comment by Shijir** [comment]
@briosheje I have tried to reproduce the same bug in Electron, but `clarity-icons` is working just fine for me. Make sure that you have no `stroke` styling on `clr-icon` svg graphic elements. 
<img width="1440" alt="screenshot 2017-05-25 09 52 07" src="https://cloud.githubusercontent.com/assets/3958008/26461107/c2e8a374-4130-11e7-81eb-a68dc7500d4b.png">

**Comment by briosheje** [comment]
@Shijir Euuuh it doesn't have any stroke.

May I ask you if you can share your package.json? maybe I'm doing something wrong there?

![schermata 2017-05-26 alle 09 10 50](https://cloud.githubusercontent.com/assets/1898880/26484568/261cb95e-41f3-11e7-93a7-1f8d62a53d93.png)

my package.json dependencies and dev dependencies:

```
"dependencies": {
    "@angular/common": "^4.0.0",
    "@angular/compiler": "^4.0.0",
    "@angular/core": "^4.0.0",
    "@angular/forms": "^4.0.0",
    "@angular/http": "^4.0.0",
    "@angular/platform-browser": "^4.0.0",
    "@angular/platform-browser-dynamic": "^4.0.0",
    "@angular/platform-server": "^4.0.0",
    "@angular/router": "^4.0.0",
    "@angular/animations": "^4.0.0",
    "@ngrx/core": "^1.2.0",
    "@ngrx/store": "^2.2.1",
    "clarity-icons":"^0.9.5-patch",
    "clarity-ui":"^0.9.5-patch",
    "clarity-angular":"^0.9.5-patch",
    "@webcomponents/custom-elements": "^1.0.0",
    "mutationobserver-shim": "^0.3.2",
    "font-awesome": "^4.7.0",
    "copy-webpack-plugin": "^4.0.1",
    "core-js": "^2.4.1",
    "electron": "^1.4.3",
    "hammerjs": "^2.0.8",
    "rxjs": "^5.2.0",
    "webpack-target-electron-renderer": "^0.4.0",
    "zone.js": "^0.8.5"
  },
  "devDependencies": {
    "@types/hammerjs": "^2.0.33",
    "@types/jasmine": "^2.2.34",
    "@types/moment-timezone": "^0.2.32",
    "@types/node": "^7.0.1",
    "@types/source-map": "^0.5.0",
    "@types/uglify-js": "^2.0.27",
    "@types/webpack": "^2.0.0",
    "angular2-template-loader": "^0.6.0",
    "awesome-typescript-loader": "^3.0.0-beta.18",
    "codelyzer": "^2.0.0-beta.2",
    "css-loader": "^0.28.0",
    "electron-packager": "^8.1.0",
    "es6-promise-loader": "^1.0.1",
    "extract-text-webpack-plugin": "^2.0.0",
    "file-loader": "^0.11.0",
    "imports-loader": "^0.7.0",
    "istanbul-instrumenter-loader": "^2.0.0",
    "jasmine-core": "^2.4.1",
    "json-loader": "^0.5.4",
    "karma": "^1.3.0",
    "karma-chrome-launcher": "^2.0.0",
    "karma-coverage": "^1.0.0",
    "karma-jasmine": "^1.0.2",
    "karma-mocha-reporter": "^2.0.0",
    "karma-remap-coverage": "^0.1.4",
    "karma-phantomjs-launcher": "^1.0.0",
    "karma-sourcemap-loader": "^0.3.7",
    "karma-webpack": "2.0.2",
    "node-sass": "^4.0.0",
    "phantomjs-polyfill": "0.0.2",
    "phantomjs-prebuilt": "^2.1.7",
    "raw-loader": "0.5.1",
    "reflect-metadata": "0.1.10",
    "remap-istanbul": "^0.9.0",
    "rimraf": "^2.5.2",
    "to-string-loader":"1.1.5",
    "sass-loader": "^6.0.1",
    "source-map-loader": "^0.2.1",
    "style-loader": "^0.16.1",
    "svg-url-loader": "^2.0.0",
    "ts-helpers": "^1.1.1",
    "tsconfig-lint": "^0.12.0",
    "tslint": "^5.0.0",
    "tslint-loader": "^3.2.0",
    "typescript": "~2.1.5",
    "url-loader": "^0.5.7",
    "webpack": "2.3.3",
    "webpack-dev-middleware": "^1.10.0",
    "webpack-dev-server": "2.4.1"
  },
```

**Comment by Shijir** [comment]
@briosheje I have installed **only** `clarity-icons` without polyfills such as these packages in `package.json`:
```
    "@webcomponents/custom-elements": "^1.0.0",
    "mutationobserver-shim": "^0.3.2",
```
You don't need these polyfills as Chromium supports custom elements out-of-the-box. So try removing the polyfills.

**Comment by briosheje** [comment]
@Shijir I've removed both the packages, but it still renders the icons in the same way.

I will investigate further and try to remove one element at a time to see what is creating issues.

Not sure whether angular 4 is actually doing some weird styling or what.

**Comment by briosheje** [comment]
@Shijir okay, I've solved the issue.

Apparently, angular 4's version ^4.0.0 had several issues with Clarity's icons, updating everything to 4.1.3 and upgrading electron to the latest version solved all the issues.

Also, I had to bring back both the polyfill dependencies to avoid unwanted warnings while compiling.

So, in a nutshell, the solution to this issue is:

1) Upgrade to angular 4 version 4.1.3 (or higher I guess).
2) Upgrade electron to version 1.6.8 or greater (I'm using 1.6.10).
